src/plugins/intel_gna/tests/deprecated/unit/inference_engine_tests/cnn_ngraph_impl_tests.cpp

```diff
@@ -763,11 +763,11 @@ TEST(CNNNGraphImplTests, ReadMeanImageFromCNNNetReader) {
     auto f = network.getFunction();
```

Comment:
Looks like this depends on new API. Do we really need to refactor tests for old API  and save old name?

Reply:
Moved from opsets to version to fix CI
Will rework these tests later

Reply:
Note, that GNA tests will be removed together with GNA before 2024.0